source/manual/create-a-local-transaction.html.md

```diff
@@ -71,7 +72,7 @@ local-links-manager-fdc5dbbb7-x7rll   2/2     Running   0          4h50m
 
 Once you've chosen a pod, connect to its Rails Console:
 ```bash
-$ gds aws govuk-integration-poweruser -- kubectl -n apps exec local-links-manager-fdc5dbbb7-vrf96 -- rails c
+$ kubectl -n apps exec local-links-manager-fdc5dbbb7-vrf96 -- rails c
```

Comment:
Small thing but omitting the `$` makes it easier to copy/paste (especially from GitHub, but also in devdocs to some extent). For example:
```suggestion
k exec local-links-manager-fdc5dbbb7-vrf96 -- rails c
```

We also have everyone set up the kubectl alias and default namespace in the getting started guide so it's safe to assume those.